ocs_ci/ocs/cluster.py

```diff
@@ -2119,6 +2119,98 @@ def is_ms_provider_cluster():
     )
 
 
+def get_osd_dump(pool_name):
+    """
+    Get the osd dump part of a given pool
+
+    Args:
+    pool_name (str): ceph pool name
+
+    Returns:
+        pool (dict): pool information from osd dump
```

Comment:
we follow the [google](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) style of python doc strings and it recommends to use varname (type): description

Reply:
> we follow the [google](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) style of python doc strings and it recommends to use varname (type): description

Above is for Args. For Returns, it should be like

```
Returns:
        bool: The return value. True for success, False otherwise.
```

in above google style of python doc strings, see example: function_with_types_in_docstring(param1, param2).

Please chnage in all places in this PR

Reply:
Done